tests/unit/common/test_hca_store.py

```diff
@@ -0,0 +1,38 @@
+import binascii
+import unittest
+
+import hca
+import numpy
+import pytest
+import zarr
+
+from matrix.common.hca_store import HCAStore
+from .. import test_bundle_spec
+
+TEST_DSS_HOST = 'https://dss.integration.data.humancellatlas.org/v1'
+
+
+class TestLambdaHandler(unittest.TestCase):
+
+    def test_hca_store_read(self):
```

Comment:
Could use more failure case tests, but would require better error handling. I'm not sure we understand this code path well enough to implement all of those. I'm OK with adding to these tests once we have v0 running.